.github/workflows/node.yml

```diff
@@ -128,52 +143,8 @@ jobs:
                   cargo-toml-folder: ./node/rust-client
               name: lint node rust
 
-    # build-macos-latest:
```

Comment:
So what is the decision with node client testing on mac? postponed?

Reply:
AFAIUI macos tests should run under the test-node job, aren't them, Avi?

Reply:
Yep